package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "dev": "node --import tsx -r dotenv/config src/dev.ts",
```

Comment:
personally I'd leave dev alone and add a `dev:watch` command with your changes, but you do you 😃